recipes/cartopy_offlinedata/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "cartopy_offlinedata" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/willirath/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: e7f3a6d47d97e133c516c631c1dbb366300c43f0dcae0d5c49c8a2a5810c477e
+
+build:
+  noarch: python
+  number: 0
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
+    - cartopy
+  run:
+    - python
+    - cartopy
+
+test:
+  imports:
+    - cartopy_userconfig
+  commands:
+    - download_cartopy_data.sh --help
+
+about:
+  home: https://github.com/willirath/cartopy_offlinedata
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'Override cartopy.config to use offline data stored in the conda env.'
+  doc_url: https://github.com/willirath/cartopy_offlinedata
+  dev_url: https://github.com/willirath/cartopy_offlinedata
+
+extra:
+  recipe-maintainers:
+    - xylar
+    - willirath
```

Comment:
@willirath, please confirm that you would be willing to be a maintainer once this feedstock is created.  (I would do most of the heavy lifting but it helps to have someone else!)

Reply:
No problem. I'm glad to contribute!